spec/services/finance/npq/course_statement_calculator_spec.rb

```diff
@@ -12,6 +12,35 @@
 
   subject { described_class.new(statement:, contract:) }
 
+  describe "#billable_declarations_count_for_declaration_type" do
+    before do
+      declarations = []
+
+      6.times do
```

Comment:
can't use `create_list` here. otherwise we end up with all declaration of the same `declaration_type` from the `sample` call